.claude/hooks/vet-go.sh

```
@@ -0,0 +1,36 @@
+# Skip non-Go files silently
+[[ "$FILE" != *.go ]] && exit 0
+
+# Skip if file doesn't exist (was deleted)
+[[ ! -f "$FILE" ]] && exit 0
+
+# Normalize to repo-relative path for consistent gopls behavior
+# This handles both absolute paths and paths relative to repo root
+REPO_ROOT=$(git rev-parse --show-toplevel 2>/dev/null)
+if [[ -n "$REPO_ROOT" && "$FILE" == /* ]]; then
+    # Convert absolute path to repo-relative
+    FILE="${FILE#$REPO_ROOT/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)